expr/ztests/record-spread-var.yaml

```diff
@@ -0,0 +1,11 @@
+zed: |
+  over a with b => ( yield {a:this+10,b} )
```

Comment:
Did you mean to use the spread operator here?

Reply:
I'll change the name...